modules/Hoot/ui/managePanel/jobs.js

```diff
@@ -49,6 +49,9 @@ export default class Jobs extends Tab {
             delete: 'delete',
             derive_changeset: 'change_history',
             upload_changeset: 'cloud_upload',
+            bulk_add: 'add_to_photos',
+            bulk_replace: 'flip_to_front',
+            bulk_differential: 'change_history',
```

Comment:
I haven't gotten my dev envi setup to actually run the bulk tasks yet, can you include a screenshot of what these job entries and icons look like?

Reply:
![image](https://user-images.githubusercontent.com/19919471/86841914-8649a700-c072-11ea-9da5-fa030b9f2523.png)

Uses same icons as the single, non-bulk, jobs